mods/sbz_progression/questbook.lua

```diff
@@ -383,12 +393,7 @@ The Automatic Filter-Injector requires 1 power to operate, it pulls out a stack
     text =
     "Craft the public storinator... it's like the regular storinator but accessible to ANYONE, regardless of protections.",
     requires = { "Storinators" }
-},
```

Comment:
You removed the secret quests? Why?

Reply:
pretty sure i didn't mean to do that, fixing